packages/react/src/runtime/ax.ts

```diff
@@ -28,8 +28,10 @@ const ATOMIC_GROUP_LENGTH = 5;
  * @param classes
  */
 export default function ax(classNames: (string | undefined | false)[]): string | undefined {
-  // short circuit if there's no class names.
-  if (classNames.length <= 1 && !classNames[0]) return undefined;
+  if (classNames.length <= 1 && (!classNames[0] || classNames[0].indexOf(' ') === -1)) {
```

Comment:
Revert `ax` back to before this [PR](https://github.com/atlassian-labs/compiled/pull/1408). So It won't have performance impact if class name compression is disabled.